taskcluster/docker/android-build/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# %ARG DOCKER_IMAGE_PARENT
```

Comment:
nit: Is this comment a leftover from testing? 🤔

Reply:
Good question! This line is actually a template shortcut used by taskgraph. It allows us to reference a parent docker image which was built in Taskcluster too.